Community.VisualStudio.Toolkit.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31321.278
+# Visual Studio Version 17
```

Comment:
It *seems* that you can only debug the `TestExtension` using VS2022 now since it references the v17 projects.  Thus the solution was updated to v17.